resources/idlharness.js

```diff
@@ -2274,6 +2285,10 @@ IdlInterface.prototype.test_interface_of = function(desc, obj, exception, expect
 IdlInterface.prototype.has_stringifier = function()
 //@{
 {
+    if (this.name === "DOMException") {
+        // Stringifier is inherited from Error
```

Comment:
(Perhaps it's just a matter of perspective, but I think from the IDL spec `Error` doesn't have a stringifier (which is an IDL concept), but does happen to inherit a `toString` function.  So perhaps mention in the comment that we return true just to avoid the caller thinking that `DOMException` incorrectly has a stringifier.)